docs/other-topics/utility-types.md

```diff
@@ -0,0 +1,98 @@
+# Utility TypeScript Types
+
+## Typing a Model Class
+
+[`ModelStatic`](pathname:///api/v7/index.html#ModelStatic) is designed to be used to type a Model *class*.
+
+Here is an example of a utility method that requests a Model Class, and returns the list of primary keys defined in that class:
+
+```typescript
+import { ModelStatic, ModelAttributeColumnOptions, Model, InferAttributes, InferCreationAttributes, CreationOptional } from '@sequelize/core';
```

Comment:
Do we want to differantiate importing functions and types?

Reply:
I don't think so. It's not related to Sequelize so it's up to them if they want to use `import type` in their codebase